src/main/java/com/ghilly/service/CountryServiceRest.java

```diff
@@ -66,4 +69,11 @@ private void exists(int countryId) {
             throw new IdIsNotFoundException("The country with this ID " + countryId + " is not found.");
         }
     }
+
+    private void throwExceptionWhenCountryNameContainsWrongSymbols(String name) {
```

Comment:
Is it about only country? Or we can to move in common class for using in advance

Reply:
And too long name

Reply:
Fixed